webapp/src/main/resources/templates/aboutUserInfo.html

```diff
@@ -0,0 +1,14 @@
+<html xmlns:sec="https://www.thymeleaf.org/thymeleaf-extras-springsecurity5">
+<nav th:fragment="nav-menu" class="menu" style="color:blue; background:skyblue; height:20px; text-align: center; ">
+    <div>Biblioteka gier planszowych</div>
+
+    <div sec:authorize="isAuthenticated()" style="color:green;">
+        Login <span sec:authentication="name"></span>
```

Comment:
bardziej chodzilo mi o to zebys w modelu sobie przekazal name z obektu Principal albo tekst 'anonymous' no ale bez roznicy wiekszej choc warto wiedziec jak sie do tego dobrac na backendzie bez htmla

Reply:
poza tym to ma byc w naglowku a nie osobnym aboutUserInfo

Endpoint -> znaczy jakas koncowka , zwykle jakis wystawionny REST sie tak okresla czyli nie \@Controller z widokiem mi chodzilo tylko stringa zwrocic na bialym tle
a to co zrobiles tu to kolejne zadanie bylo  ale w naglowku strony tym co kazalem includowac byc powinno zeby zawsze sie pokazywalo